tests/conftest.py

```diff
@@ -235,16 +241,27 @@ def fixture_run_name() -> str:
     return "fastq_run"
 
 
+@pytest.fixture(scope="function", name="original_fastqs")
+def fixture_original_fastq(fastq_dir: Path, run_name) -> CompressionData:
```

Comment:
```suggestion
def fixture_original_fastq_path(fastq_dir: Path, run_name) -> CompressionData:
```

Reply:
It is not returning a path but a CompressionData object so the path suffix is not in place here

Reply:
Changed to `original_fastq_data`